lms/djangoapps/grades/tests/test_tasks.py

```diff
@@ -109,13 +109,13 @@ def test_signal_queues_task(self, enqueue_op, test_signal):
             mock_task_apply.assert_called_once_with(args=expected_args)
 
     @patch('lms.djangoapps.grades.signals.signals.SUBSECTION_SCORE_CHANGED.send')
-    def test_subsection_update_triggers_course_update(self, mock_course_signal):
+    def test_subsection_update_triggers_signal(self, mock_subsection_signal):
         """
-        Ensures that the subsection update operation also updates the course grade.
+        Ensures that the subsection update operation triggers a signal.
         """
         self.set_up_course()
         self._apply_recalculate_subsection_grade()
-        self.assertTrue(mock_course_signal.called)
+        self.assertTrue(mock_subsection_signal.called)
```

Comment:
Why remove the validation that updating a subsection grade also updates the course grade?